### PR TITLE
Remove dependency between DocComment insertion and Completion.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Helps out with https://github.com/Microsoft/visualfsharp/issues/1825

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
Preemptively merging in.  Let me know if you want anything changed.

**Comment by Pilchie** [comment]
I'd really like @DustinCampbell to take a look at this.

File: src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs [hunk 1]
```diff
@@ -518,14 +514,6 @@ public void ExecuteCommand(ReturnKeyCommandArgs args, Action nextHandler)
                 return;
             }
 
-            // Finally, wait and see if completion is computing. If it is, we want to allow
-            // the list to pop up rather than insert a blank line in the buffer.
-            if (_completionService.WaitForComputation(args.TextView, args.SubjectBuffer))
```
Comment (Pilchie) [inline src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs]: How are we sure that this regress the scenario described in the comment?

Comment (DustinCampbell) [inline src/EditorFeatures/Core/Implementation/DocumentationComments/AbstractDocumentationCommentCommandHandler.cs]: This was very tricky at the time and the scenario is quite old (the code pre-dates the move to git). I think the overall change makes sense. We'll just need to watch out for this issue and see if it comes up again.

**Comment by CyrusNajmabadi** [comment]
Note: i manually tested this, and we have tests for this case.  Essentially, if 'enter' is hit in a doc comment *and* completion is up, then we want completion to own handling of that keystroke.

This was pretty simply done just by having completion come before doc comments.

**Comment by DustinCampbell** [comment]
Glad to hear it. I had no idea we had tests for this case. Very cool!

**Pull request merged**